template/{{.input_root_dir}}/{{template `project_name_alphanumeric_underscore` .}}/validation/notebooks/ModelValidation.py.tmpl

```diff
@@ -304,6 +310,11 @@ with mlflow.start_run(
                 )
         mlflow.log_artifact(metrics_file)
         log_to_model_description(run, True)
+        {{ if (eq .input_include_models_in_unity_catalog "yes") }}
+        # Assign "challenger" alias to indicate model version has passed validation checks
+        print("Validation checks passed. Assigning 'challenger' alias to model version.")
+        client.set_registered_model_alias(model_name, "challenger", model_version)
```

Comment:
Aliases are case-insensitive so it doesn't really matter, but stays consistent with how they look in the UI & when compared to stages.
```suggestion
        client.set_registered_model_alias(model_name, "Challenger", model_version)
```

Reply:
fair point!